polar-core/src/vm.rs

```diff
@@ -758,37 +795,46 @@ impl PolarVirtualMachine {
     }
 
     fn log(&self, message: &str, terms: &[&Term]) {
```

Comment:
This seems like it's only used once, so I'd rather use this delightfully short name for `log_with`

Reply:
I had the same thought and snipped it in #1444 !